lib/mayaUsd/ufe/UsdAttributeHolder.cpp

```diff
@@ -26,6 +27,11 @@
 #include <pxr/base/tf/token.h>
 #include <pxr/base/vt/value.h>
 #include <pxr/pxr.h>
+#include <pxr/usd/usd/editContext.h>
+#include <pxr/usd/usd/stage.h>
+
+namespace MAYAUSD_NS_DEF {
+namespace ufe {
```

Comment:
I'm a bit confused why you needed to move the function setUsdAttrMetaData (which existed inside an unnamed namespace) into the ```MayaUsd::ufe``` namespace.

Reply:
It made calling function in the namespace simpler as it avoided having to put the namespaces at each call site.

Reply:
Our coding standard recommends to use the fully qualified namespace name. I think this is a helper function which is not exported and therefore should not be in the MayaUsd::ufe namespace.